libopenage/pathfinding/pathfinder.h

```diff
@@ -226,6 +234,13 @@ class PortalNode : public std::enable_shared_from_this<PortalNode> {
 	 * Priority queue node that contains this path node.
 	 */
 	heap_t::element_t heap_node;
+
+
+	using neighbours_t = std::unordered_map<portal_id_t, int>; //I wanted to use weak_ptr as key here but couldnt get it to compile
```

Comment:
`weak_ptr` is not hashable because the contained object may expire. Also, using `portal_id_t` is probably the right call here anyway :)